content/about/introduction/introduction.html

```diff
@@ -14,166 +14,161 @@
         </p>
         <p>
-          The APG does not specify normative requirements and thus does not have a conformance model.
-          The <a href="https://www.w3.org/groups/tf/aria-practices">ARIA Authoring Practices Task Force</a>, which is part of the <a href="https://www.w3.org/groups/wg/aria">ARIA Working Group</a>,
-          uses consensus to ensure APG guidance is consistent with all relevant specifications, including ARIA, but its process for doing so is not bound by the W3C Recommendation Track process.
+          APGは規範的な要件を指定せず、したがって適合モデルを持っていません。
+          <a href="https://www.w3.org/groups/tf/aria-practices">ARIA Authoring Practices Task Force</a>は、<a href="https://www.w3.org/groups/wg/aria">ARIA Working Group</a>の一部であり、APGガイダンスがARIAを含むすべての関連仕様と一致していることを確実にするために合意を使用しますが、そのプロセスはW3Cの勧告トラックプロセスによって束縛されません。
         </p>
         <p>
-          Some of the guidance in the APG is not covered by a normative specification.
-          For example, the patterns include keyboard interaction models that combine a normative requirement specified by ARIA with conventions established by popular operating systems and software.
-          In the section on <a class="specref" href="#managingfocus">Managing focus</a>, ARIA includes normative language stating that the <kbd>Tab</kbd> key should not move focus inside of certain types of widgets.
-          However, ARIA does not specify how specific keys should move focus inside of those widgets.
-          So, the APG Task Force relies on broadly utilized keyboard conventions to provide each pattern with a list of keyboard commands for navigation and operation.
-          While those lists are not normative, the majority of such conventions are so well established that it is advisable to treat them as requirements.
-          Where that is not the case, the guidance is clearly marked as optional.
-          Note that there are some keyboard commands where users could be better served if a web widget were to dynamically adjust to mirror the users' operating system conventions.
-          It is a <a href="https://github.com/w3c/aria-practices/issues/1438">known issue</a> that some APG keyboard interaction models do not yet provide sufficient macOS-specific guidance.        </p>
+          APGのガイダンスの一部は規範的な仕様によってカバーされていません。
+          例えば、パターンには、ARIAによって指定された規範的な要件と、一般的なオペレーティングシステムおよびソフトウェアによって確立された規約を組み合わせたキーボードインタラクションモデルが含まれています。
+          <a class="specref" href="#managingfocus">フォーカスの管理</a>のセクションでは、ARIAには特定のタイプのウィジェット内で<kbd>Tab</kbd>キーがフォーカスを移動させないようにするという規範的な言語が含まれています。
+          しかし、ARIAは、これらのウィジェット内で特定のキーがどのようにフォーカスを移動させるべきかを指定していません。
+          そのため、APGタスクフォースは、広く利用されているキーボードの規約に依存して、ナビゲーションおよび操作のためのキーボードコマンドのリストを各パターンに提供します。
+          これらのリストは規範的ではありませんが、そのような規約の大部分は非常によく確立されているため、要件として扱うことが望ましいです。
+          その場合でない場合、ガイダンスは明確にオプショナルとしてマークされています。
+          注記として、ウェブウィジェットがユーザーのオペレーティングシステムの規約に動的に適応するように調整された場合、ユーザーにとってより良いサービスが提供される可能性があるキーボードコマンドがいくつかあります。
```

Comment:
（GPTが？）Noteを注記としてがんばって訳出しようとした結果、少しわかりにくくなってしまっているかなと思いました。

```suggestion
          一部のキーボードコマンドについては、ウェブウィジェットが利用者のオペレーティングシステムの規約に動的に適応するように調整されると、利用者にとってより良いサービスを提供できる場合があります。
```